doc/api/http.md

```diff
@@ -952,6 +956,10 @@ added: v5.7.0
 ### server.maxHeadersCount
 <!-- YAML
 added: v0.7.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/commit/186035243fad247e3955f
+    description: The maximum size of HTTP/1 headers is now 8KB.
```

Comment:
While there is some kind of relationship between the max header count and the total size, its not strong enough IMO to warrant this YAML block here.

I think the concern about it being hidden in a YAML block is fair.

Also, note that the max size is documented in errors. You could say those docs won't be noticed by someone reading the HTTP docs, but I would suggest that the first time someone will notice that the limit changed is when they get the error code, and when they do, they will look it up in the errors list, and see the change history.

I think it would be nice to document it in http.md as well, but its not very clear where it should be documented, which I think is why its a struggle to find a sensible place to put the YAML changes markup.

Its more work, but it might be necessary to add a section to the docs on "limits" of some kind, that discuss the various knobs and configuration related to DoS attacks. That would give a prominent place to document the header size limit.

Alternatively, you could document it under the tlsClientError event, linking to the various HTTP errors that can occur, and also calling out this specific error and the limit that will trigger it.

Reply:
Agree about someone looking up the error first probably. Still would make sense to document the change in http.md to record the change though?
Indeed I also didn't find a better place than either:
1. on the headers count which is the closest I found
2. at the top of the HTTP section before the agent as a general note about server and request

Open to other ideas.

Reply:
I'm OK with 1, with 2, or with just docing it in errors. As you wish.

Reply:
Also ok with either option.